cmd/embedded-cluster/restore.go

```diff
@@ -524,6 +578,21 @@ func restoreFromBackup(ctx context.Context, backup *velerov1.Backup, drComponent
 
 	// create a new restore object if it doesn't exist
 	if errors.IsNotFound(err) {
+		var restoreLabelSelector *metav1.LabelSelector
+		if drComponent != disasterRecoveryComponentRegistry {
```

Comment:
Nit pick: it's probably clearer to handle the positive condition here and flip the assignment. Specially when more components will be added in the future. e.g.:
```suggestion
		if drComponent == disasterRecoveryComponentRegistry {
```